### PR TITLE
adds post comment section

### PR DESCRIPTION
**Pull request by TravelByBeaut**
adds post to top of comments section

### CODE REVIEW

**Comment by cjplatten** [comment]
A few points to address: 
If i enter a user that doesn't exist it appears to still allow me to post a comment due to the optimitsic rendering, although it doesn't actually post it. It would be better if there was a more obvious indicator that the comment can be posted with an invalid user. 
I can also click the submit button multiple times for the same comment and the text stays in the input boxes.
There is also an error in the console that needs addressing!

File: src/Components/PostComment.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      return commentCopy;
+    });
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    setComments((currComment) => {
+      const commentCopy = [...currComment];
+      comment.created_at = "Just Now";
+      comment.comment_id = "new";
```
Comment (Poonam-raj) [inline src/Components/PostComment.js]: The issue with making the ID equal to 'new' is if I post two comments in a row I get a React warning telling me it's encountered two items with the same key. You need to consider a different way to give unique IDs.
See video for example of this in action

https://user-images.githubusercontent.com/61753159/186412094-781a6f7e-ca55-44a8-9d00-7f7ba2e49bd4.mov

d

File: src/Components/PostComment.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        console.log(err);
+      });
+  };
+
+  return (
+    <div className="form">
+      <form onSubmit={handleSubmit}>
+        <textarea
+          placeholder="description"
+          value={comments.body}
```
Comment (Poonam-raj) [inline src/Components/PostComment.js]: Should this line (41) be comments or comment? I feel like comments shouldn't work here as it should be an array
Not too sure about this one

**Review by Poonam-raj — APPROVED** [review]
Approved but please consider the issues I've pointed out. - I'd like you to go back and fix the ID issue before considering merging this but overall the functionality is there

**Pull request merged**